account-kit/infra/src/client/smartAccountClient.ts

```diff
@@ -144,6 +144,12 @@ export function createAlchemySmartAccountClient({
   const feeOptions =
     opts?.feeOptions ?? getDefaultUserOperationFeeOptions(chain);
 
+  // If we have a policyId, then it's safe to assume that
```

Comment:
is this true? i think it is, but i'd like confirmation.

Reply:
not really. unless they use alchmey provider, otherwise a policyId only means they have created a policy with that paymaster.

Reply:
wondering if it's actually better to include an "experimental" option (at least for now) instead of forcing everyone on to the other rpc method?

Reply:
> is it okay to do this here? the current implementation was also only including the gas manager middleware here based on the policyId, i.e.
> 
> ```ts
> ...(policyId && alchemyGasManagerMiddleware(policyId))
> ```

it is because without policyId, it doesn't make sense to involve a paymaster middleware, by any format.